test/uk/gov/hmrc/selfservicetimetopay/controllers/ArrangementControllerSpec.scala

```diff
@@ -92,7 +93,7 @@ class ArrangementControllerSpec extends UnitSpec
 
       when(arrangementConnector.submitArrangements(Matchers.any())(Matchers.any())).thenReturn(Future.successful(Right(SubmissionSuccess())))
 
-      val response = controller.submit().apply(FakeRequest("POST", "/arrangement/submit").withSession(sessionProvider.createTtpSessionId()))
+      val response = controller.submit().apply(FakeRequest("POST", "/arrangement/submit").withCookies(sessionProvider.createTtpCookie()))
```

Comment:
All test changes migrate from 'withSession' to 'withCookies' and renamed creation method